src/ildasm/dis.cpp

```diff
@@ -315,7 +315,6 @@ BOOL enumEHInfo(const COR_ILMETHOD_SECT_EH* eh, IMDInternalImport *pImport, DWOR
     COR_ILMETHOD_SECT_EH_CLAUSE_FAT ehBuff;
     const COR_ILMETHOD_SECT_EH_CLAUSE_FAT* ehInfo;
     BOOL    fTryInCode = FALSE;
-    if(g_ehInfo) VDELETE(g_ehInfo);
```

Comment:
The second thought is we might want something like this to avoid memory leak and deleting pointer without allocation.
```
if(g_ehInfo != NULL)
{
 VDELETE(g_ehInfo);
 g_ehInfo = NULL;
}
```